src/test/java/com/google/api/codegen/testdata/nodejs_test_library.baseline

```diff
@@ -565,6 +566,115 @@ describe('LibraryServiceClient', function() {
     });
   });
 
+  describe('streamShelves', function() {
+    it('invokes streamShelves without error', function(done) {
+      var client = libraryV1.libraryServiceClient();
+      // Mock request
+      var request = {};
+
+      // Mock response
+      var expectedResponse = {};
+
+      var mockStream = through2.obj(function (chunk, enc, callback) {
+        callback(null, expectedResponse);
+      });
+
+      client._streamShelves = function(actualRequest) {
+        assert.deepStrictEqual(actualRequest, request);
```

Comment:
I believe you can invoke `mockStream.write()` here, and I think that would make things clearer.